### PR TITLE
Add diversity permission

### PR DESCRIPTION
**Pull request by adamsilver**
Flows affected:

1. Onboarding flow to set up permissions
2. Organisation pages
3. Organisation edit flows
4. Invite new user
5. User pages
6. User edit flows

## Guidance for review

Main thing to check is the content inside the blue information box during the onboarding flow

### CODE REVIEW

**Comment by adamsilver** [comment]
Oh I’ve found a bug: I missed the content on step 2

**Comment by laurahermionetennant92** [comment]
Hi @adamsilver Where is this in the flow? 
Screenshot 2020-07-29 at 13 07 38

**Comment by laurahermionetennant92** [comment]
Is this text? Looks like it might need a capital V? 
Screenshot 2020-07-29 at 13 11 28

File: app/views/users/new/permissions.njk [hunk 1]
```diff
@@ -164,7 +164,7 @@
                 }
               },
               {
-                value: "See diversity information",
+                value: "view diversity information",
```
Comment (adamsilver) [inline app/views/users/new/permissions.njk:167]:
```suggestion
                value: "Access diversity information",
```

File: app/views/users/new/permissions2.njk [hunk 2]
```diff
@@ -114,7 +114,7 @@
                 }
               },
               {
-                value: "See diversity information",
+                value: "view diversity information",
```
Comment (adamsilver) [inline app/views/users/new/permissions2.njk:117]:
```suggestion
                value: "Access diversity information",
```

**Comment by laurahermionetennant92** [comment]
Hey lols @adamsilver you're there!
I'm having probs with sees and views 
think some should be Access and i'm not sure which!

**Pull request merged**